### DISCUSSION TITLE
Fixed in 1.1.8: installation fails because Demo.cs references not yet installed PrimeTween

### DISCUSSION THREAD

**Discussion by MostroW1502** [KyryloKuzyk/PrimeTween#34] [General] [answered]
Greetings!

I tried to install PrimeTween on a project that was made with 2023.2.2f1 but the installation seems to fail, as the PrimeTween namespace is unavailable when trying to reference it.
The project does have PrimeTween.Installer in the project explorer in Visual Studio.

At first i thought maybe something might've been wrong with my project, so i tried to add PrimeTween to a new Unity2d project, but the results were the same.

As i saw Unity 2019.4.40f1 mentioned in the asset store as the Unity version required, i got a work around going by downgrading the project first, installing PrimeTween and then upgrading the project again.
As i am new to Unity i am not aware if there would be an alternative way that would be preferred over the method i used?

Now that i am writing this i see that 2023.2.2f1 is not marked as LTS, is the installer only compatible with LTS version(s) or just the 2019.4.40f1 version?

**Comment by KyryloKuzyk** [comment, marked as answer]
Huge thanks for reporting the problem! I fixed the issue in version **1.1.8**.

You probably saw this error in the console that prevented PrimeTween from installing correctly:
> Assets/Plugins/PrimeTween/Demo/Scripts/Demo.cs(2,7): error CS0246: The type or namespace name 'PrimeTween' could not be found (are you missing a using directive or an assembly reference?)

The issue wasn't connected to a particular Unity version and should be fixed now.
PrimeTween is compatible with Unity 2018.4 and all newer versions, including 2023.2.2f1.

**Reply by MostroW1502** [reply]
You're most welcome ofcourse, i was unsure whether to start a discussion topic or list it as an issue, but because i thought maybe it wasn't suited for my particular version i at least made you aware of it.

The error you cited was indeed what the console showed in Unity.

Many thanks for resolving this issue!
